Resources/config/security.xml

```diff
@@ -19,7 +19,8 @@
             <argument type="service" id="security.token_storage" />
             <argument type="service" id="security.user_checker" />
             <argument type="service" id="security.authentication.session_strategy" />
-            <argument type="service" id="service_container" />
+            <argument type="service" id="request_stack" />
+            <argument type="service" on-invalid="null" /> <!-- remember_me service -->
```

Comment:
Never seen an `on-invalid="null"` when we don't have a service id. I still suggest an empty `<argument/>` and use `replaceArgument`.

Ping @OskarStark